chalicelib/checks/higlass_checks.py

```diff
@@ -478,7 +478,7 @@ def patch_expsets_processedfiles_for_higlass_viewconf(connection, **kwargs):
             # Patch the ExpSet static content
             successful_patch, patch_error =  add_viewconf_static_content_to_file(connection, expset_accession, post_viewconf_results["view_config_uuid"],
             static_content_section,
-            "tab:higlass_processed_file_testing")
+            "tab:higlass_processed_files")
```

Comment:
Maybe this should be `tab:processed-files`?

e.g. this [experiment set tab](https://data.4dnucleome.org/experiment-set-replicates/4DNESXNK2AP5/#processed-files)